docs/visualisation.rst

```diff
@@ -0,0 +1,32 @@
+Visualisation of the results
+============================
+It is quite often that the user want to visualise the results to gain
+confidence on the computed free energy. **alchemlyb** provides various
+visualisation tools to help user to judge the estimate.
+
+.. _plot_overlap_matrix:
+Overlap Matrix of the MBAR
+--------------------------
+The accuracy of the :class:`~alchemlyb.estimators.MBAR` estimator depends on
+the overlap between different lambda states. The overlap matrix from the
+:class:`~alchemlyb.estimators.MBAR` estimator could be plotted to check
+the degree of overlap. It is recommended that there should be at least
+**0.03** [Klimovich2015]_ overlap between neighboring states. ::
+
+    >>> import pandas as pd
+    >>> from alchemtest.gmx import load_benzene
+    >>> from alchemlyb.parsing.gmx import extract_u_nk
+    >>> from alchemlyb.estimators import MBAR
+
+    >>> bz = load_benzene().data
+    >>> u_nk_coul = pd.concat([extract_u_nk(xvg, T=300) for xvg in bz['Coulomb']])
+    >>> mbar_coul = MBAR()
+    >>> mbar_coul.fit(u_nk_coul)
+
+    >>> from alchemlyb.visualisation.mbar_martix import plot_mbar_omatrix
+    >>> ax = plot_mbar_omatrix(mbar_coul.overlap_maxtrix)
+    >>> ax.figure.savefig('O_MBAR.pdf', bbox_inches='tight', pad_inches=0.0)
```

Comment:
Can you include an image as an example for one of the test data? Visualization without pictures is boring.

Reply:
@orbeckst Thank you for the suggestion. I have put an image in `docs/images/O_MBAR.png` and reference to the images as `.. image:: docs/images/O_MBAR.png`. However, after I have build the doc, it doesn't seems that the image is embedded correctly (it shows the string `.. image:: docs/images/O_MBAR.png`). I wonder if you could give me a help on this please? Thank you.

Reply:
The O_MBAR.png is the generated image and I have attached the image from alchemical-analysis. Since I ported (copy and paste) the code from alchemical analysis, the images are identical.


![O_MBAR](https://user-images.githubusercontent.com/6242032/91221830-2c7e6880-e716-11ea-8d80-d46d2fc89719.png)

Reply:
Not quite sure. It works in the MDAnalysis docs (`git grep image::` or `git grep figure::`), perhaps the path needs to be different and/or defined in conf.py? Have a quick look at the sphinx docs.